tests/LinAlg/matrixTestsRajaSparseTriplet.hpp

```diff
@@ -85,10 +85,12 @@ class MatrixTestsRajaSparseTriplet : public MatrixTestsSparse
   virtual real_type getLocalElement(const hiop::hiopMatrix *a, local_ordinal_type i, local_ordinal_type j) override;
   virtual real_type getLocalElement(const hiop::hiopVector *x, local_ordinal_type i) override;
   virtual real_type* getMatrixData(hiop::hiopMatrixSparse* a) override;
+  virtual real_type getMatrixData(hiop::hiopMatrixSparse* a, local_ordinal_type i, local_ordinal_type j){assert(false);};
   virtual const local_ordinal_type* getRowIndices(const hiop::hiopMatrixSparse* a);
   virtual const local_ordinal_type* getColumnIndices(const hiop::hiopMatrixSparse* a);
   virtual local_ordinal_type getLocalSize(const hiop::hiopVector *x) override;
   virtual int verifyAnswer(hiop::hiopMatrixSparse* A, real_type answer) override;
+  virtual int verifyAnswer(hiop::hiopMatrix* A, local_ordinal_type nnz_st, local_ordinal_type nnz_ed, const double answer) {assert(false);};
```

Comment:
Provide return value to suppress compiler warnings:
```suggestion
  virtual int verifyAnswer(hiop::hiopMatrix* A, local_ordinal_type nnz_st, local_ordinal_type nnz_ed, const double answer) {assert(false); return 0;}
```